### DISCUSSION TITLE
Cleaner error message for unreachable device

### DISCUSSION THREAD

**Discussion by eikowagenknecht** [SukramJ/aiohomematic#655] [Ideas]
DONT'T DELETE THIS.

**Is your feature request related to a problem? Please describe.**
Today I couldn't turn on a light. I looked in the logs and saw this:
```
_set_value failed with ProxyException [(<Fault -1: 'Generic error (UNREACH)'>,)]: 00259D899A69EF:18, STATE, True
```

**Describe the solution you'd like**
While this is probably the technically correct way to say "Couldn't send a command to the device because it's unreachable" it would be nice to have a cleaner error message for the not so technical users.

**Describe alternatives you've considered**
---

**Additional context**
---

**Comment by SukramJ** [comment]
Um mal nur beim Thema `nicht erreichbare Geräte` zu bleiben:
- Nicht erreichbare Geräte können als "persistent notification" angezeigt werden (Blueprint).
- Entitäten von nicht erreichbare Geräten werden als unavailable gekennzeichnet.
- Es gibt einen Service der "flaky unreach" Geräte wieder reaktivieren kann (Blueprints).

Ist es wirklich sinnvoll dann auch noch für "not so technical users" eine Fehlermeldung im speziellen anzupassen?
Hört sich meiner Meinung nach nach recht viel Aufwand an um im speziellen das Thema `nicht erreichbare Geräte` zu verbessern.
